src/cfg.rs

```diff
@@ -318,7 +318,7 @@ mod tests {
 
         let cfg = CFGBuilder::build(yul::Statement::Block(block));
 
-        println!("{:?}", cfg);
+        println!("{cfg:?}");
```

Comment:
why was clippy complaining about this?

Reply:
I guess it's a new module that requires you to inline format arguments if possible.